static/beta/prod/modules/fed-modules.json

```diff
@@ -17,20 +17,20 @@
     },
     "userPreferences": {
         "manifestLocation": "/apps/user-preferences/fed-mods.json",
-        "defaultDocumentTitle": "User Preferences | User Preferences",
+        "defaultDocumentTitle": "Notification Preferences | User Preferences",
         "modules": [
             {
-                "id": "user-preferences-notifications",
+                "id": "my-notification-preferences-notifications",
                 "module": "./RootApp",
                 "routes": [
                     {
-                        "pathname": "/user-preferences"
+                        "pathname": "/my-notification-preferences"
```

Comment:
We should not change the URL as it requires additional changes in Akamai redirects. The SPUR is only about updating the titles.

Reply:
The ticket for the SPUR has a comment talking about updating the URL https://issues.redhat.com/browse/RHCLOUD-29464

Reply:
I've gone ahead and reverted the URL changes.